main.go

```diff
@@ -144,17 +173,19 @@ func main() {
 		setupLog.Error(err, "unable to register controllers to the manager")
 		os.Exit(1)
 	}
-
-	setupLog.Info("setting up webhooks")
-	if err := webhook.AddToManager(mgr, client); err != nil {
-		setupLog.Error(err, "unable to register webhooks to the manager")
-		os.Exit(1)
+	if operations["webhook"] {
+		setupLog.Info("setting up webhooks")
+		if err := webhook.AddToManager(mgr, client); err != nil {
+			setupLog.Error(err, "unable to register webhooks to the manager")
+			os.Exit(1)
+		}
 	}
-
-	setupLog.Info("setting up audit")
-	if err := audit.AddToManager(mgr, client); err != nil {
-		setupLog.Error(err, "unable to register audit to the manager")
-		os.Exit(1)
+	if operations["audit"] {
```

Comment:
This effectively deprecates turning off audit by setting the audit interval to zero. Not a blocker, but worth noting.

Reply:
ack